includes/options/class-amp-options-menu.php

```diff
@@ -205,15 +211,51 @@ public function render_theme_support() {
 				<dd>
 					<p><?php echo wp_kses_post( $reader_description ); ?></p>
 
-					<p>
-						<?php
-						/* translators: %s is the reader theme */
-						echo esc_html( sprintf( __( 'The "%s" theme is currently selected in the wizard. The ability to change it from the admin screen will come soon.', 'amp' ), AMP_Options_Manager::get_option( Option::READER_THEME ) ) );
-						?>
-					</p>
+					<div class="notice notice-info notice-alt inline">
+						<p>
+							<?php
+							/* translators: %s is the reader theme */
+							echo esc_html( sprintf( __( 'The "%s" theme is currently selected in the wizard. The ability to change it from the admin screen will come soon.', 'amp' ), AMP_Options_Manager::get_option( Option::READER_THEME ) ) );
+							?>
+						</p>
+					</div>
 
 					<?php if ( AMP_Options_Manager::get_option( Option::READER_THEME ) === get_template() ) : ?>
-						<p><?php esc_html_e( 'The currently-selected Reader theme is the same as the active theme. This means that Reader mode is disabled since it is no different from Transitional mode at this point.', 'amp' ); ?></p>
+						<div class="notice notice-info notice-alt inline">
+							<p><?php esc_html_e( 'The currently-selected Reader theme is the same as the active theme. This means that Reader mode is disabled since it is no different from Transitional mode at this point.', 'amp' ); ?></p>
```

Comment:
Should we prioritize warnings over informational notices?

Reply:
Agreed. This is just temporary since all of this will be eliminated with the work that @johnwatkins0 is doing.